apps/www/registry/default/ui/scroll-area.tsx

```diff
@@ -35,7 +35,7 @@ const ScrollBar = React.forwardRef<
       orientation === "vertical" &&
         "h-full w-2.5 border-l border-l-transparent p-[1px]",
       orientation === "horizontal" &&
-        "h-2.5 border-t border-t-transparent p-[1px]",
+        "h-2.5 flex-col border-t border-t-transparent p-[1px]",
```

Comment:
Do we need the `flex-col` here? Can this be added via the consumer?

Reply:
I believe this issue will be seen on all implementations of horizontal scroll bars, so I added it here since it seemed like bug. For a live reproduction of this issue, https://ui.shadcn.com/examples/music under `Listen Now` has a horizontal scroll bar that goes off screen when scrolling all the way to the right.